reference/enterprise-api-ref/build-api.markdown

```diff
@@ -0,0 +1,295 @@
+---
+layout: default
+title: Build API
+published: true
+tags: [reference, enterprise, API, build, modules]
+---
+
+The Build API enables you to configure modules easily to your projects.
+
+# Projects API
+
+## Create project
+
+**URI:** https://hub.cfengine.com/build/projects
+
+**Method:** POST
+
+**Parameters:**
+* **repositoryUrl** *(string)*
+  Git repository URL. Project will be synchronized with this repository. 
+  Supported protocols: `http`, `https`, `ssh`, `git`. 
+  Required.
+  Git repository URL example: https://github.com/cfengine/build-index.git
+
+* **branch** *(string)*
+  Repository branch. Required.
+
+* **name** *(string)*
+  Project name.
```

Comment:
I think it would be nice to mention what the project name defaults to if not supplied.